client/my-sites/customer-home/main.jsx

```diff
@@ -169,4 +169,4 @@ const mergeProps = ( stateProps, dispatchProps, ownProps ) => {
 
 const connectHome = connect( mapStateToProps, mapDispatchToProps, mergeProps );
 
-export default flowRight( connectHome, withTrackingTool( 'HotJar' ) )( Home );
+export default flowRight( connectHome, withTrackingTool( 'HotJar' ) )( Home );
```

Comment:
Pretty minor but the linter expects an endline at the end of files, which is why the linter failed.